pom.xml

```diff
@@ -231,7 +225,7 @@
             <dependency>
                 <groupId>org.slf4j</groupId>
                 <artifactId>slf4j-api</artifactId>
-                <version>1.7.10</version>
+                <version>1.7.21</version>
             </dependency>
```

Comment:
I suggest following the formula detailed at https://wiki.jasig.org/display/PLT/Logging+Best+Practices.  It will be consistent with other portlet projects and allow removing the <exclude> elements for log4j, etc. in dependencies.

For instance I noticed you are missing the required web.xml changes.  See above link for details.